backend/scraper/scrape_heb.py

```diff
@@ -14,8 +11,10 @@ def __init__(self):
         # Get cookies for request session
         cookies = browser.get_cookies()
         browser.quit()
+        # Creating a session
+        self.s = requests.Session()

```

Comment:
One last request, could you rename `s` to `sess` or `session` so that it's easy to tell what it is later?